### DISCUSSION TITLE
[TwigBridge] Theme custom

### DISCUSSION THREAD

**Discussion by fantomas-code** [symfony/symfony#50033] [Q&A]
**_Greetings to all 😎
How do I create a theme completely from scratch without relying on form div layout.html.twig?
I've seen that all TwigBridge themes depend on that layout.
so far I have not been able to find a tutorial that can teach me._**

**Comment by mbabker** [comment]
... I wouldn't?

That base layout takes care of a lot of core boilerplate logic for all form themes, then the framework specific form themes extend from that to add their customizations.

If you really want to start from complete scratch, just don't use the `{% use "form_div_layout.html.twig" %}` line in your base form theme.

**Reply by fantomas-code** [reply]
_**I will follow your advice. Do you know if there is any tutorial that teaches me how to create themes from scratch?**_

@mbabker
